README

```diff
@@ -4,11 +4,10 @@ integrates Asana into your web experience in the following ways:
 
   * Creates a button in your button-bar which, when clicked, pops up a
     QuickAdd window to create a new task associated with the current web page.
-    It will populate the task name with the page title by default, and
-    put the URL in the notes, along with any text you may have selected
-    when you pressed the button.
+    You can click a button to populate the task name with the page title and
+    the URL and current selected text in the notes.
 
-  * Installs the special Asana TAB+Q keyboard shortcut. When this key combo
+  * Installs the special Asana ALT+A keyboard shortcut. When this key combo
```

Comment:
If it is in fact ALT+A, that would be most unfortunate, as that key is used by certain international keyboards to write accented characters.